tests/unit_tests/models/test_openai_embeddings.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from openai import error
+
+from langchain.embeddings import OpenAIEmbeddings
+
+
+def test_azure_openai_embeddings():
+    os.environ["OPENAI_API_TYPE"] = "azure"
+    os.environ["OPENAI_API_BASE"] = "https://your-endpoint.openai.azure.com/"
+    os.environ["OPENAI_API_KEY"] = "your AzureOpenAI key"
+    os.environ["OPENAI_API_VERSION"] = "2023-03-15-preview"
+
+    embeddings = OpenAIEmbeddings(deployment="your-embeddings-deployment-name")
+    text = "This is a test document."
+
+    try:
+        query_result = embeddings.embed_query(text)
+    except error.InvalidRequestError as e:
```

Comment:
sorry what is this testing?

Reply:
That deployment_id/engine is set on openai.Embeddings when the api_type is `azure`. Basically a regression test against a similar change being made as the one that was reverted in response to this PR.